ipa-core/src/protocol/ipa_prf/aggregation/bucket.rs

```diff
@@ -51,6 +51,7 @@ impl From<MoveToBucketError> for Error {
 ///
 /// ## Errors
 /// If `breakdown_count` does not fit into `BK` bits or greater than or equal to $2^9$
+#[allow(dead_code)]
```

Comment:
Why do we need this in this PR?

Reply:
Because I haven't deleted the old aggregation code yet

Reply:
I will delete all the old-aggregation code before I merge but I would like to keep it for reference (easier to navigate code with IDE than Github). I allow the code so that all code checks still run on every push.